server/lib/redis/index.ts

```diff
@@ -1,8 +1,20 @@
 import Redis from 'ioredis'
+import logger from '../logging/logger'
 
 // TODO(tec27): provide some better wrapper around this that deals with connects/disconnects, etc.
-export default new Redis({
+const redis = new Redis({
   port: Number(process.env.SB_REDIS_PORT),
   host: process.env.SB_REDIS_HOST,
   dropBufferSupport: true,
 })
+
+redis.on('error', err => {
+  logger.error(err)
+})
+
+redis.ping().catch(err => {
+  logger.error('Error initializing Redis connection' + err)
```

Comment:
Same idea as above, it's better to pass the error on the first parameter so we get better structured data, instead of just a message string:

```ts
logger.error({ err }, 'Error initializing Redis connection')
```

I also think in this case, since this likely indicates a configuration problem, it might ne nice to use a message that hints at what can be done to fix this, e.g. `'Could not connect to Redis instance, redis host/port configuration may be incorrect'`